contributors/guide/owners.md

```diff
@@ -86,6 +86,28 @@ Instead, set a `.*` key inside `filters` (as shown in the previous example).
 **WARNING**: The `approve` plugin [does not currently respect `filters`][test-infra-7690].
 Until that is fixed, `filters` should only be used for the `labels` key (as shown in the above example).
 
+
+#### Emeritus
+
+It is inevitable, but there are times when someone may shift focuses, change jobs or step away from
+a specific area in the project for a time. These people may be domain experts over certain areas
+of the codebase, but can no longer dedicate the time needed to handle the responsibilities of
+reviewing and approving changes. They are encouraged to add themselves as an _"emeritus"_ approver
+under the `emeritus` key.
+
+Any GitHub username under the `emeritus` key will be ignored by prow for assignment. However, it can
+still be referenced by a person looking at the OWNERS file for a possible second or more informed
+opinion.
+
+When a contributor returns to being more active in that area, they can move themselves back from
+emeritus to a regular approver.
```

Comment:
I would be wary of (for example) giving someone with two years' absence carte blanche to come right back in. 
 Norms and conventions can change over time.

I feel like it should be at the discretion of other approvers.  If that's too hand-wavy / what would help me make this decision as an approver would be something like: proof of recent reviews but with reduced quantity/duration vs someone coming to approver through the reviewer role.

Reply:
Approvers making that call was more so my intent, but didn't explicitly call it out. I'll clean up the language.